tests/integration/test_password_rotation.py

```diff
@@ -33,7 +33,8 @@ async def test_deploy_active(ops_test: OpsTest):
         charm,
         application_name=APP_NAME,
         num_units=3,
-        resources={"zookeeper-image": "ubuntu/zookeeper:latest"},
+        resources={"zookeeper-image": "jardon/zookeeper:3.6.3"},
```

Comment:
Where did you and Jon land on where to put the dockerhub owner? Or is it up in the air for now?

Reply:
we havent had a discussion.  @delgod do you have any insight on the above comment from Jon?

Reply:
Answer on my comment below.